### PR TITLE
Remove quantified postconditions on Set operations

### PR DESCRIPTION
**Pull request by romac**
Closes: #881

### CODE REVIEW

File: frontends/benchmarks/verification/valid/Iterables.scala [hunk 1]
```diff
@@ -3,41 +3,78 @@ import stainless.lang._
...
 
+  @extern @pure
+  def setFilterAxiom[A](set: Set[A], p: A => Boolean, a: A): Unit = {
+    ()
+  }.ensuring { _ =>
+    if (set.contains(a) && p(a)) {
+      set.filter(p).contains(a)
+    } else {
+      !set.filter(p).contains(a)
+    }
```
Comment (jad-hamza) [inline frontends/benchmarks/verification/valid/Iterables.scala:56]: Can this be written as `(set.contains(a) && p(a)) == set.filter(p).contains(a)`?

**Comment by vkuncak** [comment]
Oops, I ended up redoing some of this in #1000 ... We should check.

**Comment by vkuncak** [comment]
Ok, so after #1000, all that remains is to port `frontends/benchmarks/verification/valid/SetAxioms.scala` to have call the alternatively named postcondition axioms, to have some extra tests. The other changes are no longer needed.

**Comment by vkuncak** [comment]
After checking some more, this is in fact addressed already in master.

**Pull request closed without merging**